wazo_webhookd/services/helpers.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2017 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import contextlib
+import logging
+
+import requests
+
+logger = logging.getLogger(__name__)
+
+
+class HookExpectedError(Exception):
+    def __init__(self, detail):
+        self.detail = detail
+        super(HookRetry, self).__init__()
+
+
+class HookRetry(Exception):
+    def __init__(self, detail):
+        self.detail = detail
+        super(HookRetry, self).__init__()
+
+
+@contextlib.contextmanager
+def requests_automatic_hook_retry(task):
+    try:
+        yield
+    except requests.exceptions.HTTPError as exc:
+        if exc.response.status_code == 410:
+            logger.info("http request fail, service is gone ({}/{}): "
```

Comment:
There are 5 `{}` and 6 arguments
`%s` should also be used instead of `.format` to allow the logger to skip string formatting if the string does not get logged.

Reply:
I just saw that this was probably copied from the celery_tasks.py module since you use `%s` in other files.

Reply:
Cool, I put format because I see format everywhere, but year for logging doesn't make sense.